container/prod/configure.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# run as the www-data user. no access to su or sudo
+set -ex
+
+# this should be redundent, but just in case
+cd /srv/journal-cms/web
+
+# determine if the site is installed. 
+# if it looks like it's installed we can skips some steps
+rm -f site-was-installed.flag
+site_installed=false
+if ../vendor/bin/drush cget system.site name; then
+    touch site-was-installed.flag
+    site_installed=true
+fi
+
+if ! $site_installed; then
+    ../vendor/bin/drush site-install config_installer -y
+    redis-cli flushall
```

Comment:
this errors out for me, because it talks to `127.0.0.1` whereas it should probably be talking to `journal-cms_redis_1`?

Reply:
or the service name: `redis`

Reply:
the configuration should be using the hostnames set up by docker-compose, which are `redis` and `mysql` (I think).

configuration files are in `./config/drupal-container.settings.php`